code/modules/jobs/job_types/security.dm

```diff
@@ -126,6 +126,7 @@ Detective
 	minimal_player_age = 7
 	exp_requirements = 300
 	exp_type = EXP_TYPE_CREW
+	special_notice = "You are not a security officer. Do not do their job."

```

Comment:
The Detective kinda is expected to help the Officers in most situations. I think it's a bit silly to give them a baton, cuffs and a gun and then tell them not to help the people he works with constantly.

Reply:
`You are not a security officer, do not do their job for them. You should assist them if they need help, however.`

Reply:
New one is slightly different from Ziiro's proposed change.